fs/_util.ts

```diff
@@ -1,6 +1,22 @@
 // Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
 import * as path from "../path/mod.ts";
-import { basename, fromFileUrl, normalize } from "../path/mod.ts";
+import { basename, normalize } from "../path/mod.ts";
+
+/**
+ * Test whether `src` and `dest` resolve to the same location
+ * @param src src file path
+ * @param dest dest file path
+ */
+export function isSamePath(
+  src: string | URL,
+  dest: string | URL,
+  sep: string = path.sep,
```

Comment:
The 3rd parameter looks only used in testing. Do we really need this?

Reply:
(We usually don't need to specify platform specific params explicitly because they are abstracted by path module)

Reply:
> The 3rd parameter looks only used in testing. Do we really need this?

Idealy not. I had some trouble producing consistent results in the tests involving mixed URL and string params using win32 paths. It's probably just a testing issue, and I will look for a better solution in the meantime.

Reply:
Today I discovered the `ignore` option and fixed the issue :)